strax/corrections.py

```diff
@@ -75,6 +75,28 @@ def list_corrections(self):
         database = self.client[self.database_name]
         return [x['name'] for x in database.list_collections()]
 
+    def read_by_index(self, correction, when):
```

Comment:
the naming is not descriptive of what the functions is doing. The function will return the closest item in time before and after the given date. That is true whether the time field is indexed on mongodb or not. I would rename to "read_at" or something that would help a maintainer (or you in a year from now) quickly understand what the function does without reading the code.

Reply:
good suggestion